src/components/NcSelect/NcSelect.vue

```diff
@@ -1010,27 +1009,33 @@ export default {
 		},
 
 		propsToForward() {
-			const {
-				// Props handled by this component
-				inputClass,
-				noWrap,
-				placement,
-				userSelect,
-				// Props to forward
-				...initialPropsToForward
-			} = this.$props
-
+			const vueSelectKeys = [
+				...Object.keys(VueSelect.props),
+				...VueSelect.mixins.flatMap(mixin => Object.keys(mixin.props ?? {})),
+			]
+			const initialPropsToForward = Object.fromEntries(
+				Object.entries(this.$props)
+					.filter(([key, _value]) => vueSelectKeys.includes(key)),
+			)
 			const propsToForward = {
 				...initialPropsToForward,
 				// Custom overrides of vue-select props
 				calculatePosition: this.localCalculatePosition,
 				filterBy: this.localFilterBy,
 				label: this.localLabel,
 			}
-
 			return propsToForward
 		},
 	},
+
+	mounted() {
+		if (!this.inputLabel && !this.ariaLabelCombobox) {
+			Vue.util.warn('[NcSelect] An `inputLabel` or `ariaLabelCombobox` should be set. This warning can be ignored if an external label for `inputId` is already set.')
+		}
+		if (this.inputLabel && this.ariaLabelCombobox && this.inputLabel !== this.ariaLabelCombobox) {
+			Vue.util.warn('[NcSelect] Only one of `inputLabel` or `ariaLabelCombobox` should to be set. Or they should be set to the same string.')
+		}
+	},
```

Comment:
What about making the same as with `*Fields`: add `(input)labelOutside` boolean props to explicitly tell that there is a label.

Reply:
And then remove the default value from `ariaLabelCombobox`. The current default value makes little sense - user cannot understand what "options" mean in any context, and it "hides" the issue. Because of the default value, all the current `NcSelect` will work without a warning like a valid select, while they should have a warning and be fixed.

I'd not consider it a breaking change, because there will be no visible changes or breaks in apps.

Reply:
It'd also save from requirement to always manually set `:aria-label-combobox="null"` like in examples in this PR

Reply:
Added `labelOutside` + removed `ariaLabelCombobox` default string

> I'd not consider it a breaking change, because there will be no visible changes

Depends how you define *breaking* but good with me!

Reply:
> Depends how you define _breaking_ but good with me!

Breaking = existing selects may break the current UI on update.

Formally removing `aria-label-combobox` default value also might be considered breaking. But the previous value didn't make sense anyway. So I think it is safe to get rid of it.